src/generateSdk/sdkGenerator/TsSdkGenerator.ts

```diff
@@ -292,7 +292,10 @@ class SdkGenerator implements SdkGeneratorInterface {
 
                 // @ts-expect-error A refactor need to be performed here to avoid this error
                 const methodView: MethodViewType = {
-                    name: methodDefinition.name,
+                    // methods start with lowercase
+                    name:
+                        methodDefinition.name.charAt(0).toLowerCase() +
+                        methodDefinition.name.slice(1),
```

Comment:
Why is this change needed? What if i want to have functions that start with upper case? It's a ts convention to have lowercase named methods, but we should not impose it.

Reply:
This is needed because all methods in Go will start with an uppercase letter.
The client code for our templates in ts/js expects methods starting with lowercase. We need this for a full-stack template with a go server and a ts client to work.

Reply:
@costinsin i added the root property in the AstGeneratorInput class. much cleaner now